backend/.env.example

```
@@ -40,6 +40,10 @@ MINIO_PORT=9000
 MINIO_ROOT_USER=adminuser
 MINIO_ROOT_PASSWORD=password123
 MINIO_DEFAULT_BUCKET=fallout-bucket
+# Public URL for MinIO (used for generating public file URLs)
+# Local dev: http://localhost:9000
+# Production: https://media.yourdomain.com (your public subdomain)
+MINIO_PUBLIC_URL=https://media.yourdomain.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_PUBLIC_URL key should go before the MINIO_ROOT_PASSWORD key

(UnorderedKey)